app/assets/javascripts/application.js

```diff
@@ -16,3 +16,13 @@
 //= require_tree .
 
 $(function(){ $(document).foundation(); });
+
+function toggleVisibility(id) {
+  var e = document.getElementById(id);
+  if(e.style.display == 'block')
+  {
+     e.style.display = 'none';
+  else
```

Comment:
Expected an identifier and instead saw 'else'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.